cdap-ui/app/cdap/components/Wizard/index.js

```diff
@@ -46,8 +46,23 @@ export default class Wizard extends Component {
       activeStep: this.props.wizardConfig.steps[0].id,
       success: false,
       loading: false,
-      error: ''
+      error: '',
+      inputIsEmpty: false
     };
+
+    if (this.props.wizardType === "PublishPipeline") {
```

Comment:
`Wizard` as a component should not be aware of `PublishPipeline` There is a `PublishPipeline` component under `CaskWizard`. That should be the right place to fix this issue. Not here.